pkgs/by-name/pr/prettier-plugin-go-template/package.nix

```diff
@@ -17,6 +17,13 @@ buildNpmPackage rec {
 
   npmDepsHash = "sha256-PpJnVZFRxpUHux2jIBDtyBS4qNo6IJY4kwTAq6stEVQ=";
 
+  dontNpmPrune = true;
+
+  postInstall = ''
+    cd "$nodeModulesPath"
+    find -mindepth 1 -maxdepth 1 -type d | grep -Exv "\./(ulid|prettier)" | xargs rm -rf
+  '';
```

Comment:
```suggestion
  # please add a comment explaining why this is needed
  postInstall = ''
    pushd "$nodeModulesPath"
    find -mindepth 1 -maxdepth 1 -type d -print0 | grep --null-data -Exv "\./(ulid|prettier)" | xargs -0 rm -rfv
    popd
  '';
```

Reply:
I'm not really sure, I searched on GitHub how others got it working and found [this](https://github.com/42LoCo42/obscura/blob/28a5f8377bab82895c9bc4f20ac8738cbb17caa1/packages/prettier-plugin-go-template.nix#L12). 

Apart from ulid and prettier those are just dev dependencies.

Reply:
My proposed change only deals with proper escaping of files with spaces in them. For the comment you can describe the error you'd see without the fix